ipet/parsing/ReaderManager.py

```diff
@@ -363,13 +393,11 @@ def fromXMLFile(xmlfilename):
 
     @staticmethod
     def processXMLElem(elem):
+        # FARI1 what if not? rm undefined?
```

Comment:
If the element is not a reader manager, this method throws a "ReferencedBeforeAssignment" error at the return.